src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -441,7 +441,7 @@ local function process_function(fn)
     local cparam = string.format('arg%u', j)
     local param_type = real_type(param[1])
     local lc_param_type = real_type(param[1]):lower()
-    local extra = ((param_type == "Object" or param_type == "Dictionary") and "false, ") or ""
+    local extra = (param_type == "Object" and "true, ") or (param_type == "Dictionary" and "false, ") or ""
```

Comment:
@bfredl Is this the right way to do this to allow `{command}` to take a Lua function?

Reply:
oops probably should have read the entire thread before commenting :D . This does work though . I think it's fine as this only effecta lua api bindings .